forms-flow-nav/src/Navbar.jsx

```diff
@@ -170,9 +170,21 @@ const NavBar = React.memo(({ props }) => {
 
   useEffect(() => {
     fetchSelectLanguages((data) => {
-      setSelectLanguages(data);
+      const tenantdata = JSON.parse(StorageService.get("TENANT_DATA"));
+      const userLanguageList = (MULTITENANCY_ENABLED && tenantdata?.details.langList) || USER_LANGUAGE_LIST ;
+      const getSupportedLanguages = (data) => {
+        let userLanguagesArray = [];
+        if (typeof userLanguageList === 'object') {
+           userLanguagesArray = Object.values(userLanguageList);
```

Comment:
try to optimize this code really need to write a function there ? `getSupportedLanguages`

Reply:
@shuhaib-aot optimized.